Firestore/core/src/firebase/firestore/util/comparator_holder.h

```diff
@@ -47,7 +47,7 @@ class ComparatorHolder {
 template <typename C>
 class ComparatorHolder<C, true> : private C {
  protected:
-  explicit ComparatorHolder(const C&) noexcept {
+  explicit ComparatorHolder(const C& /* comparator */) noexcept {
```

Comment:
Maybe put the parameter name outside of comment like
`explicit ComparatorHolder(const C& comparator) noexcept {`

Reply:
That causes a different warning about an unused parameter. This way both the compiler and the linter are happy.

I could also do
```c++
explicit ComparatorHolder(const C& comparator) noexcept {
  (void)comparator;
}
```
But considered this to be inferior. I have no strong preference though.